### PR TITLE
[FIX] Fix documentation and a bug in NCE Block

### PR DESCRIPTION
**Pull request by eric-haibin-lin**
## Description ##
The concat order of the NCE block output is reversed. 

### Essentials ###
- [x] PR's title starts with a category (e.g. [BUGFIX], [MODEL], [TUTORIAL], [FEATURE], [DOC], etc)
- [x] Changes are complete (i.e. I finished coding on this PR)
- [x] All changes have test coverage
- [x] Code is well-documented

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by mli** [comment]
Job PR-558/1 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-558/1/index.html

File: src/gluonnlp/model/sampled_block.py [hunk 1]
```diff
@@ -91,7 +91,7 @@ def hybrid_forward(self, F, x, sampled_values, label, w_all, b_all):
             label_vec = F.reshape(label, (-1, 1))
             new_label_sampled = F.zeros_like(pred_sampled)
             new_label_true = F.ones_like(label_vec)
-            new_label = F.Concat(new_label_sampled, new_label_true, dim=1)
+            new_label = F.Concat(new_label_true, new_label_sampled, dim=1)
```
Comment (szha) [inline src/gluonnlp/model/sampled_block.py]: What models does this change affect?

Comment (eric-haibin-lin) [reply src/gluonnlp/model/sampled_block.py]: language model trained with NCE. gluonnlp doesn't provide any script using NCE.

**Comment by mli** [comment]
Job PR-558/3 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-558/3/index.html

**Comment by mli** [comment]
Job PR-558/6 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-558/6/index.html

**Comment by mli** [comment]
Job PR-558/7 is complete. 
Docs are uploaded to http://gluon-nlp-staging.s3-accelerate.dualstack.amazonaws.com/PR-558/7/index.html

**Pull request merged**